code/_onclick/item_attack.dm

```diff
@@ -34,7 +34,7 @@
 	user.lastattacked = M
 	M.lastattacker = user
 
-	M.attacked_by(src, user)
+	. = M.attacked_by(src, user)

```

Comment:
aaaa fuck why is this code butts, I need to check if the spear actually hit them or if it got blocked first so you can't impale past shields even if the damage got blocked